src/Illuminate/Foundation/resources/exceptions/renderer/components/syntax-highlight.blade.php

```diff
@@ -1,39 +1,59 @@
 @props([
     'code',
-    'grammar',
-    'lightTheme' => 'light-plus',
-    'darkTheme' => 'dark-plus',
-    'withGutter' => false,
+    'language',
+    'editor' => false,
     'startingLine' => 1,
     'highlightedLine' => null,
     'truncate' => false,
 ])
 
-@use('Phiki\Phiki')
-@use('Phiki\Grammar\Grammar')
-@use('Phiki\Theme\Theme')
-@use('Phiki\Transformers\Decorations\GutterDecoration')
-@use('Phiki\Transformers\Decorations\LineDecoration')
-@use('Phiki\Transformers\Decorations\PreDecoration')
-
 @php
-    $highlightedCode = (new Phiki)->codeToHtml($code, $grammar, ['light' => $lightTheme, 'dark' => $darkTheme])
-        ->withGutter($withGutter)
-        ->startingLine($startingLine)
-        ->decoration(
-            PreDecoration::make()->class('bg-transparent!', $truncate ? ' truncate' : ''),
-            GutterDecoration::make()->class('mr-6 text-neutral-500! dark:text-neutral-600!'),
-        );
+    $fallback = $truncate ? '<pre class="truncate"><code>' : '<pre><code>';
+    $lines = explode("\n", $code);
 
-    if ($highlightedLine !== null) {
-        $highlightedCode->decoration(
-            LineDecoration::forLine($highlightedLine)->class('bg-rose-200! [&_.line-number]:dark:text-white! dark:bg-rose-900!'),
-        );
+    if ($editor) {
+        foreach ($lines as $index => $line) {
+            $lineNumber = $startingLine + $index;
+            $highlight = $highlightedLine === $index;
+            $lineClass = implode(' ', [
+                'block px-4 py-1 h-7 even:bg-white odd:bg-white/2 even:dark:bg-white/2 odd:dark:bg-white/4',
+                $highlight ? 'bg-rose-200! dark:bg-rose-900!' : '',
+            ]);
+            $lineNumberClass = implode(' ', [
+                'mr-6 text-neutral-500! dark:text-neutral-600!',
+                $highlight ? 'dark:text-white!' : '',
+            ]);
+
+            $fallback .= '<span class="' . $lineClass . '">';
+            $fallback .= '<span class="' . $lineNumberClass . '">' . $lineNumber . '</span>';
+            $fallback .= htmlspecialchars($line);
+            $fallback .= '</span>';
+        }
+
+    } else {
+        $fallback .= htmlspecialchars($code);
     }
+
+    $fallback .= '</code></pre>';
```

Comment:
When JavaScript is disabled, the fallback will be rendered without syntax highlighting.

<img width="1335" height="474" alt="Image" src="https://github.com/user-attachments/assets/dfda0b05-8fe8-4a0a-aa4d-f2c709ba5c1f" />

<img width="1333" height="479" alt="Image" src="https://github.com/user-attachments/assets/3a281c66-238c-4f14-840e-bc8acb7a6a15" />